ports/mimxrt/Makefile

```diff
@@ -277,9 +312,10 @@ ifeq ($(MICROPY_FLOAT_IMPL),double)
 $(LIBM_O): CFLAGS := $(filter-out -Wdouble-promotion -Wfloat-conversion, $(CFLAGS))
 endif
 
-SRC_SS += $(MCU_DIR)/gcc/startup_$(MCU_SERIES).S
-
-SRC_S += shared/runtime/gchelper_m3.s \
+SRC_SS = \
+	$(MCU_DIR)/gcc/startup_$(MCU_SERIES).S \
+	hal/resethandler_MIMXRT10xx.S
+SRC_S = shared/runtime/gchelper_m3.s \
```

Comment:
this `SRC_S` line didn't need to change (should still be +=)

also please retain the blank line between them

Reply:
fixed.